docs/news.d/ace8_tao4.rst

```diff
@@ -0,0 +1,6 @@
+.. news-prs: none
+.. news-start-section: Platform Support and Dependencies
+.. news-start-section: ACE/TAO
+- Added support for building with `ACE8.0.0/TAO4.0.0 <https://github.com/DOCGroup/ACE_TAO/releases/tag/ACE%2BTAO-8_0_0>`
```

Comment:
```suggestion
- Added support for building with `ACE 8.0.0 / TAO 4.0.0 <https://github.com/DOCGroup/ACE_TAO/releases/tag/ACE%2BTAO-8_0_0>`
```

I don't know how it will render with RST but it looks like it could use some space